tests/components/gios/test_config_flow.py

```diff
@@ -57,11 +45,10 @@ async def test_invalid_station_id(hass):
 
 async def test_invalid_sensor_data(hass):
     """Test that errors are shown when sensor data is invalid."""
-    with patch("gios.Gios._get_stations", return_value=VALID_STATIONS), patch(
-        "gios.Gios._get_station", return_value=VALID_STATION
-    ), patch("gios.Gios._get_station", return_value=VALID_STATION), patch(
-        "gios.Gios._get_sensor", return_value={}
-    ):
+    with patch("gios.Gios._get_stations", return_value=STATIONS), patch(
```

Comment:
its best to patch these where they are imported. these global patches can cause issues in random contexts

Reply:
Good point, fixed.